### ISSUE TITLE
Not able to access renderer or representation drop-downs and their options using keyboard

### ISSUE THREAD

**Issue by bojanrajkovic** [Microsoft/workbooks#136]
### Steps to Reproduce

1. Launch a workbook
2. Run 2+2
3. Attempt to use the keyboard to reach the renderer selection next to the result

### Expected Behavior

Can reach the renderer selection dropdown and choose a renderer via the keyboard.

### Actual Behavior

Cannot reach renderer selection drop-down.

The same behavior happens if you run code like `new object { foo = bar }` and try to switch between ToString and object members representations.

#### Xamarin Workbooks
| Detail | Value |
| :- | :- |
| Version | 0.0.0-local |
| Git Branch | @PACKAGE_HEAD_BRANCH@ |
| Git Hash | @PACKAGE_HEAD_REV_SHORT@ |
| VSTS Definition | @PACKAGE_BUILD_HOST_LANE@ |

#### System Info
| Component | Value |
| :- | :- |
| Windows | Microsoft Windows NT 10.0.16299.0 (64-bit) |
| CPU Cores | 8 |
| Physical Memory | 16 GB |

#### System-Installed Software
| Component | Version |
| :- | :- |
| Xamarin Simulator | 1010103100 |

#### VisualStudio.15.Release Components
| Component | Version |
| :- | :- |
| Visual Studio Enterprise 2017 | 15.5.1+27130.2003 |
| Xamarin iOS | 11.6.1-2 |
| Xamarin Android | 8.1.0-24 |

#### VisualStudio.15.Preview Components
| Component | Version |
| :- | :- |
| Visual Studio Enterprise 2017 | 15.6.0-pre.1.0+27205.0 |
| Xamarin iOS | 11.4.0-214 |
| Xamarin Android | 8.1.0-24 |

### Accessibility Tags

MAS36

**Comment by bojanrajkovic** [comment]
Did a little more digging, and it turns out that Monaco can trivially be told _not_ to eat tabs (and this behavior is fungible via a keybinding at runtime).

When using a screen reader (Narrator, probably also JAWS/NVDA), Monaco actually tells you how to disable that behavior, which is great. We probably should link to Monaco's accessibility documentation from our own somewhere.

**Comment by bojanrajkovic** [comment]
In addition to needing to disable Monaco's tab-eating behavior, we need to also make sure that all elements can be reached by keyboard and that screen readers correctly recognize them.
